src/context/Context.tsx

```diff
@@ -0,0 +1,21 @@
+import { createContext, useContext, useState } from "react";
+
+type initial = {
+    login:boolean | undefined;
+    setLogin: (active: boolean|undefined) => void;
+}
+const initialState:initial = {
+    login: undefined,
+    setLogin: () => null
+}
+
+
+const Context = createContext(initialState);
+
+export const AuthProvider = ({ children }: { children: React.ReactNode | any }) => {
```

Comment:
좀 지식자랑 같은 코멘트지만 이게 좀더 편하실 거예요

```suggestion
export const AuthProvider: React.FC = ({ children }) => {
```

Reply:
아하 벨로퍼트분께서 React.FC는 안쓰는 걸 추천한다길래,,, 근데 children이 디폴트로 들어가 있으니 이럴 때 편하네요!

Reply:
ㅋㅋㅋㅋㅋ 상당한 논쟁거리입니다